### DISCUSSION TITLE
Ableton Folder like display mode

### DISCUSSION THREAD

**Discussion by ghost** [SynthstromAudible/DelugeFirmware#566] [Ideas]
If you're familiar with Ableton, you might know about its "FOLDER" feature. It's a display mode that, when activated, only shows the notes in a sequence that have triggers, effectively hiding those rows that don't have any active notes. This helps in reducing the need to scroll between the highest and lowest notes, streamlining the workflow especially for complex sequences.

In my opinion, implementing a similar feature in the Deluge would be incredibly useful, providing an organized and focused view, particularly when dealing with dense arrangements.

Potential shortcut to activate this mode could be a combination of **SHIFT** + **KEYBOARD**

Would love to hear feedback from the community and see if this is something that others would find valuable too.

**Comment by sichtbeton** [comment]
Yep, very good proposition. Its sometimes hard to keep up where notes are.
